contentcuration/contentcuration/frontend/shared/data/serverSync.js

```diff
@@ -232,12 +237,15 @@ function syncChanges() {
             console.warn('There was an error during syncing with the backend for', err); // eslint-disable-line no-console
           });
       });
+    })
+    .then(() => {
+      syncActive = false;
```

Comment:
Do we also need to handle the `.catch()` path?

Reply:
All the errors should already have been caught in the previous catches, so this is just chaining from there.

Reply:
But it probably doesn't hurt...